### PR TITLE
Actualize readme

### PR DESCRIPTION
**Pull request by MCJOHN974**
#1180

### CODE REVIEW

**Comment by MCJOHN974** [comment]
@Graur, please, have a look

**Comment by MCJOHN974** [comment]
I also tested all other code snippets from readme, they work

File: README.md [hunk 1]
```diff
@@ -177,10 +177,10 @@ This is how you iterate:
...
 
 ```
-2 x 2 = 4
-3 x 3 = 9
-4 x 4 = 16
-5 x 5 = 25
+2x2 = 4
+3x3 = 9
+4x4 = 16
+5x5 = 25
```
Comment (Graur) [inline README.md:183]: @MCJOHN974 Is this the real output of code above? It seems there should still be spaces between multiplications

**Comment by MCJOHN974** [comment]
@Graur hm, yes, I just checked it again and old output with spaces was correct, I fixed it

**Comment by MCJOHN974** [comment]
But, btw, current code from readme has no spaces in ``sprintf``

**Review by Graur — APPROVED** [review]
@MCJOHN974 Good one!
@yegor256 Please, check

**Pull request merged**